### ISSUE TITLE
Geometry analysis - distance calculation

### ISSUE THREAD

**Issue by janash** [MolSSI-Education/python_scripting_cms#33]
We never introduce `**` syntax for raising to a power, but use it in a distance calculation exercise.

**Comment by janash** [comment]
For the solution, we should separate out the different calculations, as it seemed to help students understand what was going on at GLRM.

ie, in the `for` loop:

~~~
x_distance = atomA_coords[0] - atomB_coords[0]
y_distance = atomA_coords[1] - atomB_coords[1]
z_distance = atomA_coords[2] - atomB_coords[2]

distance = numpy.sqrt(x_distance**2 + y_distance**2 + z_distance**2)

~~~

**Comment by janash** [comment]
Addressed by #37
